subprojects/core/src/main/java/org/gradle/internal/fingerprint/impl/DefaultGenericFileTreeSnapshotter.java

```diff
@@ -57,7 +57,8 @@ public void visitFile(FileVisitDetails fileDetails) {
                     new DefaultFileMetadata(
                         FileType.RegularFile,
                         fileDetails.getLastModified(),
-                        fileDetails.getSize()
+                        fileDetails.getSize(),
+                        false
```

Comment:
Why doesn't this check if the file is a symlink here?

Reply:
Because it is not yet implemented. I'll implement this as part of #13053.